src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
```

Comment:
Aside from public methods, you can also try and include Javadoc header comments for your classes perhaps?